src/components/PlayerIndicator.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+import { Player, State } from "../../public/store";
+import { useSelector } from "react-redux";
+
+const PlayerIndicatorContainer = styled.div``;
+const PlayerContainer = styled.div``;
+
+const PlayerIndicator: React.FC = () => {
+  const thisPlayer: Player = useSelector((state: State) => state.thisPlayer);
+  const currentPlayer: Player = useSelector(
+    (state: State) => state.currentPlayer
+  );
+  return (
+    <PlayerIndicatorContainer>
+      <PlayerContainer>
+        <h2>Your name: {thisPlayer.name}</h2>
+        <h2>Your team: {thisPlayer.team}</h2>
+      </PlayerContainer>
+      <PlayerContainer>
+        <h2>Turn: {currentPlayer.name}</h2>
+        <h2>Team: {currentPlayer.team}</h2>
+      </PlayerContainer>
+    </PlayerIndicatorContainer>
+  );
+};
+
+export default PlayerIndicator;
```

Comment:
(not on this line, but this is the closest place to add a note. Scroll down for the real thing):

```
Check warning on line 5 in lib/lobby.test.ts

GitHub Actions
/ build (14.x)
lib/lobby.test.ts#L5
Title must be a string
```

Definitely turn this rule off in your linter configs. After AJ showed me that you don't need a string in the title of a describe() block, I have been using this trick everywhere :D